CMakeLists.txt

```diff
@@ -136,7 +136,9 @@ set(double_conversion_INCLUDE_DIR ${CMAKE_CURRENT_LIST_DIR}/vendor/double-conver
 target_include_directories(${json_library_TARGET} PUBLIC ${double_conversion_INCLUDE_DIR})
 target_link_libraries(${json_library_TARGET} double-conversion)
 
-option(SPOTIFY_JSON_BUILD_TESTS "Build tests and benchmarks" ON)
+if (NOT DEFINED SPOTIFY_JSON_BUILD_TESTS)
```

Comment:
Is it not possible to set options from the including CMake file?

Also, our CMake style says no space between `if` and `(`.